atecc608a_se.c

```diff
@@ -0,0 +1,228 @@
+#include "atecc608a_se.h"
+
+#include "atca_status.h"
+#include "atca_devtypes.h"
+#include "atca_iface.h"
+#include "atca_command.h"
+#include "atca_basic.h"
+#include "atca_helpers.h"
+
+/* Uncomment to print results on success */
+//#define DEBUG_PRINT
+
+#ifdef DEBUG_PRINT
+#include <stdio.h>
+#endif
+#include <stdbool.h>
+
+#define ATCAB_INIT()                                        \
+    do                                                      \
+    {                                                       \
+        if (atcab_init(&atca_iface_config) != ATCA_SUCCESS) \
+        {                                                   \
+            status = PSA_ERROR_HARDWARE_FAILURE;            \
+            goto exit;                                      \
+        }                                                   \
+    } while(0)
+
+#define ATCAB_DEINIT()    \
+    do                    \
+    {                     \
+        atcab_release();  \
```

Comment:
Realistically, what errors might `atcab_release` return? If it fails, should we report this as a `HARDWARE_FAILURE`?

Reply:
It might return ATCA_BAD_PARAM if there's no global device initialized, and an implementation of hal might return an error if an i2c device cannot be released. The implementations that I could find never return anything else than success.

Reply:
Ok. Please add this information in a comment somewhere to justify ignoring errors from `atcab_release`.